src/Plugin_Command.php

```diff
@@ -1474,7 +1500,15 @@ private function get_details( $file ) {
 		return $plugin_folder[ $plugin_file ];
 	}
 
+	/**
+	 * Performs deletion of plugin files
+	 *
+	 * @param $plugin - Plugin fetcher object (name, file)
+	 * @return bool - If plugin was deleted
+	 */
 	private function delete_plugin( $plugin ) {
+		do_action( 'delete_plugin', $plugin->file );
+
```

Comment:
Didn't want to `// phpcs:ignore ` as that would remove all checks. It's not clear to me how I would understand which custom matchers these match to as they have "names" but nothing matches names or comments in https://github.com/wp-cli/extension-command/blob/main/phpcs.xml.dist:

```
FILE: src/Plugin_Command.php
-------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
-------------------------------------------------------------------------------------------------------------------------
 1510 | ERROR | Hook names invoked by a theme/plugin should start with the theme/plugin prefix. Found: "delete_plugin".
 1534 | ERROR | Hook names invoked by a theme/plugin should start with the theme/plugin prefix. Found: "deleted_plugin".
-------------------------------------------------------------------------------------------------------------------------
```

Reply:
Ah, nm, found it: https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/WordPress/Sniffs/NamingConventions/ValidHookNameSniff.php

Reply:
maybe...still not sure why phpcs doesn't just provide the class name of the sniff that failed

Reply:
🤷 https://github.com/search?q=repo%3AWordPress%2FWordPress-Coding-Standards+%22Hook+names+invoked+by+a+%22&type=code

Reply:
The error you wanna ignore is `WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound`

I isually use `vendor/bin/phpcs --report=xml` to find that out quickly.

Reply:
Ah, turns out you need to use `-s` to see the source of the phpcs rules